benchcab/benchcab.py

```diff
@@ -225,7 +227,11 @@ def checkout(self, config_path: str):
         rev_number_log = ""
 
         for model in self._get_models(config):
-            model.repo.checkout()
+            try:
+                model.repo.checkout()
+            except Exception:
+                self.logger.error("Try using `benchcab clean realisations` first")
+                sys.exit(1)
```

Comment:
Ideally, I don't want to add any `sys.exit(1)` in the code, I'd rather have exceptions raised.

Reply:
Yeah, it was just for not having the stack trace here that the user doesn't want to know. But I think it's better to add the appropriate `Exception` here as mentioned.